lm_admin_guide.md

```diff
@@ -198,7 +198,10 @@ help                  Show help
 ### A note about volumes
 
 Docker Compose uses Docker volumes for data storage. These will persist
-between start and stop actions. Use the regular Docker commands to delete
+between start and stop actions. Database schema will be automacally updated 
```

Comment:
"automacally" -> "automatically"

Reply:
Fixed in https://github.com/crs4/life_monitor/pull/134/commits/31741e0589a4ae0dadafd31f6d05068ae303f68d